src/welcomeEmail/welcomeEmail.js

```diff
@@ -0,0 +1,46 @@
+function sendEmail(){
+  var dataSheet = SpreadsheetApp.getActiveSpreadsheet().getSheetByName("coderbunker join in button");
+  
+  for (var i = 2; i <= dataSheet.getLastRow(); i++){
+    var name = dataSheet.getRange(i, 1).getValue();
+    var emailAddress = dataSheet.getRange(i, 2).getValue();
+    var status = dataSheet.getRange(i, 5).getValue();
+    
+    
+    if (status == '') {  // if there is no sending record
+    // sending onboarding email
+    var htmlTemplate = HtmlService.createTemplateFromFile("EmailTemplate");
+    htmlTemplate.name = name;
+    var htmlContent = htmlTemplate.evaluate().getContent();
+    var subject = "Welcome " + name + "!";
+ 
+      var LogoUrl = "http://secure.meetupstatic.com/photos/event/d/3/6/f/600_451134127.jpeg";
```

Comment:
Upload the image to our Github repo or Google drive and change the image's link. In case someday this link expired.

Reply:
Uploaded the image in Github repo in branch "issue67-welcome-email", will change the image's link in script code after merging branch. (can use this link now, but it's more safe to use the image when it is in master branch, especially there could be a gap between merging branch and update the link. 
https://github.com/coderbunker/gas/blob/issue67-welcome-email/src/welcomeEmail/coderbunker.jpg?raw=true )

Reply:
OK